1_fetch/src/fetch_streamflow.R

```diff
@@ -0,0 +1,15 @@
+fetch_streamflow <- function(ind_file, sites_yml, dates_yml, stream_params_yml, gd_config) {
+
+  dates <- yaml::yaml.load_file(dates_yml) # these dates are user input right now but should be dynamic based on storm
+  pcodes <- yaml::yaml.load_file(stream_params_yml)
+  site_ids <- yaml::yaml.load_file(sites_yml)
```

Comment:
I think we should pass these in as the already loaded yamls. Then we won't have to do this every time. Something like: https://github.com/USGS-R/nawqa_wqp/blob/master/1_wqpdata.yml#L30-L36